rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
 profile = "minimal"
-channel = "nightly-2022-08-05"
+channel = "nightly-2023-08-05"
```

Comment:
Have you tried a newer version? like 2023-11-01.

Reply:
The latest version that Nix supports is 2023-10-01 and it works fine.